README.md

```diff
@@ -43,6 +67,8 @@ kind: App
 metadata:
   name: pyroscope
   namespace: giantswarm
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
```

Comment:
Why ?

Reply:
Without this label, the app is not picked up and deployed by the `app-operator` on CAPI clusters

Reply:
Sure but his version means it's deployd on the MC

Reply:
I updated this :)